src/main/java/io/spaship/operator/service/k8s/Operator.java

```diff
@@ -37,414 +36,413 @@
+        var phase = pod0.getStatus().getPhase();
+        var readiness = Readiness.isPodReady(pod0);
+        LOG.debug("pod phase is {} ", phase);
+        return phase.equalsIgnoreCase("Running") && readiness;
+
+    }
+
+    // TODO remove if blocks
+    private void processK8sList(KubernetesList result, UUID tracing, String nameSpace) {
+
+        var eb = EventStructure.builder().uuid(tracing.toString());
+
+        result.getItems().forEach(item -> {
+            if (item instanceof Service svc) {
+                LOG.debug("creating new Service in K8s, tracing = {}", tracing);
+                ocClient.services().inNamespace(nameSpace).createOrReplace(svc);
+                eb.websiteName(item.getMetadata().getLabels().get(WEBSITE))
+                        .environmentName(item.getMetadata().getLabels().get(ENVIRONMENT))
+                        .state("service created");
+            }
+            if (item instanceof Deployment dep) {
+                LOG.debug("creating new Deployment in K8s, tracing = {}", tracing);
+                ocClient.apps().deployments().inNamespace(nameSpace).createOrReplace(dep);
+                eb.websiteName(item.getMetadata().getLabels().get(WEBSITE))
+                        .environmentName(item.getMetadata().getLabels().get(ENVIRONMENT))
+                        .state("deployment created");
+            }
+            if (item instanceof StatefulSet sfs) {
+                LOG.debug("creating new Deployment in K8s, tracing = {}", tracing);
+                ocClient.apps().statefulSets().inNamespace(nameSpace).createOrReplace(sfs);
+                eb.websiteName(item.getMetadata().getLabels().get(WEBSITE))
+                        .environmentName(item.getMetadata().getLabels().get(ENVIRONMENT))
+                        .state("StatefulSet created");
+
+            }
+            if (item instanceof PersistentVolumeClaim pvc) {
+                LOG.debug("creating new pvc in K8s, tracing = {}", tracing);
+                ocClient.persistentVolumeClaims().inNamespace(nameSpace).createOrReplace(pvc);
+                eb.websiteName(item.getMetadata().getLabels().get(WEBSITE))
+                        .environmentName(item.getMetadata().getLabels().get(ENVIRONMENT))
+                        .state("pvc created");
+            }
+            if (item instanceof Route route) {
+                LOG.debug("creating new Route in K8s, tracing = {}", tracing);
+                ocClient.routes().inNamespace(nameSpace).createOrReplace(route);
+                eb.websiteName(item.getMetadata().getLabels().get(WEBSITE))
+                        .environmentName(item.getMetadata().getLabels().get(ENVIRONMENT))
+                        .state("route created");
+            }
+            if (item instanceof ConfigMap cm) {
+                LOG.debug("creating new ConfigMap in K8s, tracing = {}", tracing);
+                ocClient.configMaps().inNamespace(nameSpace).createOrReplace(cm);
```

Comment:
Note : There might be a potential issue of exception, can we handle it here ! (Optional)

Reply:
Okay @arkaprovob